pkg/console/console.go

```diff
@@ -34,7 +41,7 @@ func Run() error {
 
 	}
 
-	serviceCert, err := LoadCertificates(serviceCertPath, serviceKeyPath)
+	serviceCert, err := tlsconfig.LoadCertificates(serviceCertPath, serviceKeyPath)
```

Comment:
This `serviceCert` is used for the `tokenSigningKey` and will not be rotated on changes to the `serviceCert`. Can you make sure it is rotated too?

Reply:
done